config/message.go

```diff
@@ -0,0 +1,52 @@
+// format for messages : ZFSOrder and ZFSResponse
+
+package config
+
+import (
+	"io/ioutil"
+)
+
+type ZFSOrder struct {
```

Comment:
__[golint]__
* [golint] exported type ZFSOrder should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->